tests/CommunityToolkit.Mvvm.DisableINotifyPropertyChanging.UnitTests/Test_DisableINotifyPropertyChanging.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using CommunityToolkit.Mvvm.ComponentModel;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace CommunityToolkit.Mvvm.UnitTests.Configuration;
+
+[TestClass]
+public class Test_DisableINotifyPropertyChanging
+{
+    static Test_DisableINotifyPropertyChanging()
```

Comment:
Shouldn't this use `[ClassInitialize]`, and reset the switch on `[ClassCleanup]` so it can't affect other tests?

Reply:
The switch cannot be reset, as it's used to set a static read-only field 😉
This test file is in a separate, dedicated unit test project specifically to test this. Look at the file oath, you'll see it's not in the same unit test project as the others 🙂